src/bootstrap.rs

```diff
@@ -0,0 +1,273 @@
+// Copyright 2015 MaidSafe.net limited
+//
+// This MaidSafe Software is licensed to you under (1) the MaidSafe.net Commercial License,
+// version 1.0 or later, or (2) The General Public License (GPL), version 3, depending on which
+// licence you accepted on initial access to the Software (the "Licences").
+//
+// By contributing code to the MaidSafe Software, or to this project generally, you agree to be
+// bound by the terms of the MaidSafe Contributor Agreement, version 1.0, found in the root
+// directory of this project at LICENSE, COPYING and CONTRIBUTOR respectively and also
+// available at: http://www.maidsafe.net/licenses
+//
+// Unless required by applicable law or agreed to in writing, the MaidSafe Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, either express or implied.
+//
+// See the Licences for the specific language governing permissions and limitations relating to
+// use of the MaidSafe Software.
+
+use time;
+use cbor;
+use cbor::CborTagEncode;
+use rustc_serialize::{Decodable, Decoder, Encodable, Encoder};
+use sodiumoxide::crypto;
+use std::net;
+use std::net::SocketAddr;
+use transport::Endpoint;
+use std::fs::File;
+use std::io;
+use std::io::prelude::*;
+use std::path;
+use std::env;
+use beacon;
+
+type BootStrapContacts = Vec<Contact>;
+
+static MAX_LIST_SIZE: usize = 1500;
+
```

Comment:
I asked here @BSMaidSafe lets see if we get some better mechanism here http://stackoverflow.com/questions/29784502/convert-rust-vectors-to-arrays-and-back